packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -111,23 +108,17 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
     entryPoints['main'] = [mainPath];
   }
 
-  const differentialLoadingMode = buildOptions.differentialLoadingNeeded && !buildOptions.watch;
   if (platform !== 'server') {
-    if (differentialLoadingMode || tsConfig.options.target === ScriptTarget.ES5) {
-      if (buildBrowserFeatures.isEs5SupportNeeded()) {
-        const polyfillsChunkName = 'polyfills-es5';
-        entryPoints[polyfillsChunkName] = [path.join(__dirname, '..', 'es5-polyfills.js')];
-
-        if (!buildOptions.aot) {
-          if (differentialLoadingMode) {
-            entryPoints[polyfillsChunkName].push(path.join(__dirname, '..', 'jit-polyfills.js'));
-          }
-          entryPoints[polyfillsChunkName].push(path.join(__dirname, '..', 'es5-jit-polyfills.js'));
-        }
-        // If not performing a full differential build the polyfills need to be added to ES5 bundle
-        if (buildOptions.polyfills) {
-          entryPoints[polyfillsChunkName].push(path.resolve(root, buildOptions.polyfills));
-        }
+    if (buildBrowserFeatures.isEs5SupportNeeded()) {
```

Comment:
Do we still need this logic?

Reply:
No but this PR was getting large and there are a variety of ES5 special cases throughout the code and tests that we'll need to review as well.